app/javascript/flavours/polyam/styles/components/status.scss

```diff
@@ -620,7 +620,7 @@
   .icon {
     width: 14px;
     height: 14px;
-    position: relative;
+    vertical-align: middle;
```

Comment:
This unfortunately doesn't really fix the alignment.
Haven't checked glitch flavour yet though.